Removing unavailable videos part 2 (#10623)
mdn/content: files/en-us/mozilla/add-ons/webextensions/your_second_webextension/index.md

        manifest.json

> Now install the extension temporarily. Open "about:debugging" in Firefox, click "Load Temporary Add-on", and select your manifest.json file. You should then see the extension's icon appear in the Firefox toolbar:

Reviewer: "Now install the extension temporarily." doesn't sound quite right to me. Maybe

```suggestion
Now load the extension as a temporary add-on. Open "about:debugging" in Firefox, click "Load Temporary Add-on", and select your **manifest.json** file. You should then see the extension's icon appear in the Firefox toolbar:
```
